bittensor/metagraph.py

```diff
@@ -499,6 +499,12 @@ def sync(
 
                 subtensor = bittensor.subtensor(network='archive')
         """
+
+        if block > 300:
+            raise ValueError(
```

Comment:
Can we use `bittensor.logging.error` here?  Can still raise a value error, but should be logged this way as well

Reply:
Looks like we're getting test errors now